src/components/SoundCloudPlayer.vue

```diff
@@ -143,6 +143,28 @@
     },
     methods: {
       /**
+       * Skip to a specific location
+       * when the progress bar is clicked
+       */
+      progressClicked(e){
+        let prog = this.$refs.prog;
+        console.log(prog);
+        let x = e.pageX - prog.offsetLeft - 326, // or e.offsetX (less support, though)
+            y = e.pageY - prog.offsetTop,  // or e.offsetY
+            clickedValue = x * prog.max / prog.offsetWidth;
+        this.song.currentPosition = clickedValue;
+        this.player.seekTo(clickedValue);
+      },
+      /**
+       * Need to delay getting info
+       * for 1 second after iFrame is loaded
+       * to prevent async issues
+       */
+      iFramePreLoad(){
+        let vm = this;
```

Comment:
You are correct. I had that in there for debugging reasons, didn't realize it was committed.